runtime/compiler/control/CompilationThread.cpp

```diff
@@ -7519,7 +7519,18 @@ TR::CompilationInfoPerThreadBase::preCompilationTasks(J9VMThread * vmThread,
          }
       }
 #if defined(J9VM_OPT_JITSERVER)
-   bool cannotDoRemoteCompilation = cannotPerformRemoteComp(
+   TR_Debug *debug = TR::Options::getDebug();
+   bool dontRemoteCompile = false;
+   if(debug)
```

Comment:
You can use directly `if (TR::Options::getDebug())` since debug is not used downstream